modifications/specials.json

```diff
@@ -146,7 +140,7 @@
     },
     "special_inertial_impact": {
         "id": 32,
-        "edname": "special_inertial_impact",
```

Comment:
This change looks curious. So which experimental effect does not exist?

Reply:
Distortion Field does not exist, `special_distortion_field` points to Inertial Impact